hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpServer.java

```diff
@@ -153,6 +154,7 @@ public class HttpServer implements FilterContainer {
   public static final String SPNEGO_PROXYUSER_FILTER = "SpnegoProxyUserFilter";
   public static final String NO_CACHE_FILTER = "NoCacheFilter";
   public static final String APP_DIR = "webapps";
+  public static final String HTTP_UI_SHOW_STACKTRACE_KEY = "hbase.ui.showStackTraces";
```

Comment:
Let's follow the naming pattern from other properties and refrain from using came case in property names.

Reply:
Thank you @wchevreuil , I have addressed this now.